packages/lockfile-lint/src/config.js

```diff
@@ -52,6 +52,11 @@ module.exports = (argv, exitProcess = false, searchFrom = process.cwd()) => {
           "validates that the resource URL specifies the same package name as that listed as the lockfile entry's key",
         implies: 'allowed-hosts'
       },
+      i: {
+        alias: ['validate-integrity'],
```

Comment:
To be more precise we're not validating the integrity itself here but rather it's type, so wdyt about updating the flag and naming convention to be explicit about the type `validate-integrity-sha256` ?

Reply:
@lirantal 
I guess this one was not resolved. It was merged after I changed from `--validate-integrity` to `--validate-integrity-sha512`
I'll open a new PR and revert back to `validate-integrity` during the weekend. 
Unless you want to reconsider using `validate-integrity-type`.

Reply:
Either way works. Whichever you think is best.